.golangci.yml

```diff
@@ -1,13 +1,25 @@
 run:
```

Comment:
this `.golangci.yml` is actually not needed any more

Reply:
the log shows that the golint is enabled but replaced with revive